ci_build/azure_pipelines/keras2onnx_application_tests.yml

```diff
@@ -111,7 +111,7 @@ jobs:
         INSTALL_KERAS_RESNET: pip install keras-resnet
         INSTALL_TRANSFORMERS: pip install transformers==3.4.0
         INSTALL_NUMPY: pip install numpy==1.19.0
-        NIGHTLY_BUILD_TEST: python run_all.py --exclude "test_keras_applications_v2.py test_mask_rcnn.py"
+        NIGHTLY_BUILD_TEST: python run_all_v2.py --exclude "test_keras_applications_v2.py test_mask_rcnn.py"
```

Comment:
I guess the reason that some keras tests failed is in the below error [message](https://dev.azure.com/tensorflow-onnx/tensorflow-onnx/_build/results?buildId=17463&view=logs&j=517fe804-fa30-5dc2-1413-330699242c05&t=c7d0fcb4-3a10-5b26-dfea-e879a93fc2ab)?

```
/home/vsts/work/_temp/696c7bc6-ddd1-44a6-b34d-3903fc84df87.sh: line 16: INSTALL_NUMPY: command not found
```